aiohttp/pytest_plugin.py

```diff
@@ -224,9 +224,13 @@ def pytest_pyfunc_call(pyfuncitem):  # type: ignore[no-untyped-def]
     """Run coroutines in an event loop instead of a normal function call."""
     fast = pyfuncitem.config.getoption("--aiohttp-fast")
     if inspect.iscoroutinefunction(pyfuncitem.function):
-        existing_loop = pyfuncitem.funcargs.get(
-            "proactor_loop"
-        ) or pyfuncitem.funcargs.get("loop", None)
+        existing_loop = (
+            pyfuncitem.funcargs.get("proactor_loop")
+            or pyfuncitem.funcargs.get("selector_loop")
+            or pyfuncitem.funcargs.get("uvloop_loop")
```

Comment:
@webknjaz adding `uvloop_loop` as unused arg is necessary as here the loop to be used is selected based on func args.

Reply:
Oh, I see. Then, it's better to change this to be based on markers and add marker as a decorator.

Reply:
Let's not change any behaviour of the plugin. We're deprecating it, so let's just try and keep the behaviour the same and write the tests in prepration for them being run through pytest-aiohttp.